presto-product-tests/src/main/java/com/facebook/presto/tests/cli/PrestoCliTests.java

```diff
@@ -144,7 +144,8 @@ public void shouldRunBatchQuery()
         assertThat(trimLines(presto.readRemainingOutputLines())).containsAll(nationTableBatchLines);
     }
 
-    @Test(groups = CLI, timeOut = TIMEOUT)
+    //@Test(groups = CLI, timeOut = TIMEOUT)
```

Comment:
Oh. This issue(https://github.com/prestodb/tempto/issues/117) is already fixed. I missed that.